extensions/wikia/Chat2/js/controllers/ChatBanModalLogs.js

```diff
@@ -1,29 +1,33 @@
-var ChatBanLogController = function () {
+/**
+ * Bootstrap the app
+ */
+$(function () {
+	'use strict';
+
 	$('.chat-change-ban').click(function (event) {
-		var title = $.msg('chat-ban-modal-change-ban-heading'),
-			userId = $(this).data('user-id'),
-			okCallback = function (expires, reason) {
-				$.post(wgScript + '?action=ajax&rs=ChatAjax&method=blockOrBanChat', {
-						userToBanId: userId,
-						time: expires,
-						reason: reason,
-						mode: 'global',
-						token: mw.user.tokens.get('editToken')
-					},
-					function () {
-						window.location.reload();
-					}
-				);
-			},
-			chatBanModal = new ChatBanModal(title, okCallback, {'isChangeBan': true, userId: userId});
+		var userId = $(this).data('user-id');
+
+		require(['chat-ban-modal'], function (chatBanModal) {
+			chatBanModal.open(
+				$.msg('chat-ban-modal-change-ban-heading'),
+				function (expires, reason) {
```

Comment:
Minor comment: only in favor of readability I'd consider extracting the callback function.